introduction.md

```diff
@@ -1,3 +1,10 @@
 My name is Jesse.  I'm about kickin' ass, I'm about takin' names, and we're about makin' games.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?